search-app/docs/api.md

```diff
@@ -1413,7 +1413,9 @@ Find collections with tag_data in the form of tag_data[tag_key]=tag_value. It fi
 Collections can be found by searching for associated variables. The following variable parameters are supported.
 
 * variable_name
-  * options: pattern
+  * supports `pattern`
```

Comment:
I changed my mind. I put in a change to index variable-name as its own fields under collection so that searching by variable-name would have the same options as searching by measurement. I removed variables nested mapping from collections as that is really future code for handling searching multiple fields in a variable. We can add it back in when we have that need.